src/content/1.7/modules/creation/external-services.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Interacting with APIs
+---
+
+Interacting with APIs
+=====================
```

Comment:
```suggestion
# Interacting with APIs
```

Reply:
This should be done on the whole project, as I used another page as a template.